### PR TITLE
Updated version from 2.204.2 to 2.222.1

### PR DESCRIPTION
**Pull request by kevin-dsouza**

### CODE REVIEW

File: init/master-runcmd.cfg [hunk 1]
```diff
@@ -25,6 +25,7 @@ runcmd:
   - sh /opt/wait_for_setup_done.sh
   - mkdir /var/lib/jenkins/plugin-updates
   - mkdir /var/lib/jenkins/plugin-updates/archive
+  - wget http://localhost:8080/jnlpJars/jenkins-cli.jar -P /var/cache/jenkins/war/WEB-INF/ -auth "admin:$(cat /var/lib/jenkins/secrets/initialAdminPassword)"
```
Comment (kevin-dsouza) [inline init/master-runcmd.cfg:28]: This is what pull the cli from the right place and puts it in the existing path

**Comment by claydanford** [comment]
@kevin-dsouza, Does this work as expected during upgrades and clean installs?

Can you bump the version number for this release, in all places?

**Comment by kevin-dsouza** [comment]
> @kevin-dsouza, Does this work as expected during upgrades and clean installs?
> 


@claydanford Yes Ive tested it by deploying 2.204 then upgrading to 2.222.1 and also directly deployed 2.222.1

> Can you bump the version number for this release, in all places?
I did a search and replace on the old version number, did I miss something?

**Comment by claydanford** [comment]
@kevin-dsouza Yes, the module version number which is currently 3.2.1. Please bump the minor version number.

**Comment by kevin-dsouza** [comment]
> @kevin-dsouza Yes, the module version number which is currently 2.3.1. Please bump the minor version number.

Gotcha ...done

**Pull request merged**